server/api_grammar.hpp

```diff
@@ -62,6 +63,14 @@ template <typename Iterator, class HandlerT> struct APIGrammar : qi::grammar<Ite
                     qi::double_)[boost::bind(&HandlerT::addCoordinate, handler, ::_1)];
         hint = (-qi::lit('&')) >> qi::lit("hint") >> '=' >>
                stringwithDot[boost::bind(&HandlerT::addHint, handler, ::_1)];
+        timestamp = (-qi::lit('&')) >> qi::lit("t") >> '=' >>
```

Comment:
ok, fair enough. IIRC, the reasoning back then was that garbage in -> garbage out. I think we can leave that as is.